README.md

```diff
@@ -3,78 +3,58 @@
+Terminus Plugin that contains a collection of commands useful during the build step on a [Pantheon](https://www.pantheon.io) site that manages its files using Composer, and uses a GitHub PR workflow with Behat tests run via Circle CI (or some other testing service). For detailed set-up instructions, see the [Terminus Build Tools Guide](https://pantheon.io/docs/guides/build-tools/).
 
-An [example circle.yml file](example.circle.yml) has been provided to show how this tool should be used with CircleCI. When a test runs against a "canonical" repository on GitHub, the following things will happen:
+## Requirements
 
-- Git is configured for making clones and commits.
-- Terminus 1.x is installed.
-- The oldest multidev testing environments are deleted.
-- A build step is fired off via `composer build-assets`.
-- A new multidev environment is created for testing.
-- The build artifacts are pushed up to the test environment.
+- If you are using Terminus 2, you must use the unstable Build Tools 2.x release
+- If you are using Terminus 1, you must use the stable Build Tools 1.x release
 
-The multidev environment created to test the pull request persists until the pull request is merged. Pantheon on-server development (SFTP) mode may be used as usual on these environments; any commits made on the Pantheon dashboard will be pushed back to the GitHub repository on the PR branch. Once the PR is merged to the master branch, then the corresponding multidev environment is also merged into the Pantheon dev environment. When using this workflow, all work is done in pull requests; the dev environment is never used for development.
+PHP 7.2 is recommended.
 
-See below for the list of supported commands. This plugin is only available for Terminus 1.x.
-
-## Setup
-
-In order to use this plugin, you will need to set up a GitHub repository and a CircleCI project for the site you wish to build. Credentials also need to be set up. Most of the work can be done for you automatically using the New Project Quickstart below, or you may set everything up manually. You may need to install this Terminus plugin first by going to the [Installation Section](#installation) of this document.
-
-### Credentials
-
-In order to use the build:project:create command, the first thing that you need to do is set up credentials to access your Git and CI Providers. Instructions on creating these credentials can be found on the pages listed below:
-
-- GitHub: https://help.github.com/articles/creating-an-access-token-for-command-line-use/
-- Bitbucket: https://confluence.atlassian.com/bitbucket/app-passwords-828781300.html
-- Circle CI: https://circleci.com/account/api
-- GitLab: https://gitlab.com/profile/personal_access_tokens
-
-The GitHub token needs the "repo" and "delete repo" scopes.
+### Installing Build Tools 2.x:
+```
+mkdir -p ~/.terminus/plugins
+composer create-project -d ~/.terminus/plugins pantheon-systems/terminus-build-tools-plugin:^2
```

Comment:
@mcdwayne is reporting this is not working. Maybe since the tagged version, e.g. `2.0.0-beta6`, is beta? I'm not sure if `—stability=dev` will help.

Perhaps we should keep this up to date with the latest beta release, e.g. `composer create-project -d ~/.terminus/plugins pantheon-systems/terminus-build-tools-plugin:2.0.0-beta6`, until we have a stable `2.x` release, then update to `terminus-build-tools-plugin:^2`

Reply:
You're right, this needs to be specified as `^2.0.0-beta6`. Keeping it up-to-date is optional. It's useful to do this as documentation (I do this in the docker container so that one can see at a glance that an older version could not have been included), but ^2.0.0-beta6 will still pick up a beta7 or later if available.

Reply:
Another issue to report. 
According to this doc: https://github.com/pantheon-systems/terminus-build-tools-plugin/blob/fb04bbc755566dca8feacdcb16c35a9fc912eee4/DEVELOPMENT.md
"GITLAB_TOKEN A generated GitLab token - needs at least api and read_user scopes."

But when creating a token with both api and user_read, the token looks like this formulation:
xxxxxxxxxx-_xxxxxxxx
This causes the following error:
"GitLab authentication tokens should be 20-character strings containing only the letters a-z and digits (0-9). Please enter your token again."

Solution:
Creating a GitLab token with only API scope returns a token with the following formulation:
xxxxxxxxxxxxxxxxxxxx
Which works fine.

Reply:
read_user is not needed, but we should fix that limitation. We'll do that as a separate issue, https://github.com/pantheon-systems/terminus-build-tools-plugin/issues/178